### PR TITLE
Move v4.1 release updates to master

### PR DESCRIPTION
**Pull request by grantfirl**
This PR should follow #213 and #214 (it contains those changes as well). Its contents have been reviewed before as part of the v4.1 release. This PR is merely merging those changes (primarily python3 and build stuff) back into master.

Update on 2020/11/13:
The namelist variables iovr_lw and iovr_sw were changed to iovr in the GFS_v16beta and HWRF suites. This was supposed to happen with #213, but I missed this.

### CODE REVIEW

**Comment by climbfuji** [comment]
I am going to review this after the previous PRs have been merged. It's hard to look for the differences otherwise.

**Comment by grantfirl** [comment]
> I am going to review this after the previous PRs have been merged. It's hard to look for the differences otherwise.

OK, @climbfuji, this should be easier to review now that it has the previous changes merged in.

File: scm/src/CMakeLists.txt [hunk 1]
```diff
@@ -51,12 +51,19 @@ include(CMakeForceCompiler)
...
-  MESSAGE(STATUS "The NETCDF environment variable is $ENV{NETCDF}")
-  SET(CMAKE_PREFIX_PATH ${CMAKE_PREFIX_PATH} $ENV{NETCDF})
-ELSE(DEFINED ENV{NETCDF})
-  MESSAGE(FATAL_ERROR "The NETCDF environement variable must be set to point to your NetCDF installation before building. Stopping...")
-ENDIF(DEFINED ENV{NETCDF})
+#If using Hera, use the FindNetCDF.cmake module to find the NetCDF installed with NCEPLIBS; 
+#otherwise, rely on the NETCDF environment variable to point to the desired NetCDF installation
+if("$ENV{CMAKE_Platform}" STREQUAL "hera.intel")
+  MESSAGE(STATUS "Using FindNetCDF.cmake module for $ENV{CMAKE_Platform}")
+  FIND_PACKAGE(NetCDF REQUIRED COMPONENTS C CXX Fortran)
```
Comment (climbfuji) [inline scm/src/CMakeLists.txt:58]: We should clean this up at some point - all systems should be able to use the cmake FIND_PACKAGE functionality.

**Review by climbfuji — APPROVED** [review]

File: scm/src/CMakeLists.txt (hunk 1 above)
Comment (grantfirl) [reply scm/src/CMakeLists.txt:58]: Yes. Other systems are relying on users to set an environment variable at the moment.

**Pull request merged**